.pytool/Plugin/UncrustifyCheck/UncrustifyCheck.py

```diff
@@ -575,26 +575,28 @@ def _process_uncrustify_results(self) -> None:
         self._formatted_file_error_count = len(formatted_files)
 
         if self._formatted_file_error_count > 0:
-            logging.error(
+            logging.warning('Uncrustify found {0} files with formatting errors'.format(self._formatted_file_error_count))

```

Comment:
I tried to use f-strings elsewhere in the file. Do you think you could use those here for consistency?